generators/server/templates/src/main/java/package/web/rest/AccountResource.java.ejs

```diff
@@ -80,6 +81,24 @@ public class AccountResource {
             throw new AccountResourceException("User could not be found");
         }
     }
+
+    /**
+     * {@code GET  /authenticate} : check if the user is authenticated, and return its login.
+     *
+     * @param request the HTTP request.
+     * @return the login if the user is authenticated.
+     */
+    @GetMapping("/authenticate")
+    <%_ if (reactive) { _%>
+    public Mono<String> isAuthenticated(ServerWebExchange request) {
```

Comment:
can we have a unit or an integration test on this plz ?

Reply:
sure. I will check if we have an existing test case as this API is already exposed in `Session`, `JWT` authentication scenarios.